cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -142,7 +144,8 @@ bool CobTwistController::initialize()
     {
         twist_controller_params_.frame_names.push_back(chain_.getSegment(i).getName());
     }
-    register_link_client_ = nh_.serviceClient<cob_srvs::SetString>("obstacle_distance/registerLinkOfInterest");
+
+    register_link_client_ = nh_.serviceClient<cob_srvs::SetString>("/register_links");
```

Comment:
This is clearly related to #101 - which is still WIP
Try to separate everything related to this feature from the TwistController-Extensions feature, i.e. JointSingularityAvoidance